cli/tests/test_log_command.rs

```diff
@@ -1280,6 +1280,16 @@ fn test_graph_styles() {
     ○  initial
     ◆
     "###);
+
+    // Invalid style name
+    let stderr = test_env.jj_cmd_failure(
+        &repo_path,
+        &["log", "--config-toml=ui.graph.style='unknown'"],
+    );
+    insta::assert_snapshot!(stderr, @r###"
+    Config error: enum GraphStyle does not have variant constructor unknown
+    For help, see https://github.com/martinvonz/jj/blob/main/docs/config.md.
```

Comment:
Should this point to https://martinvonz.github.io/jj/latest/config/ instead?

Reply:
Done in https://github.com/martinvonz/jj/pull/4393